buzzmobile/process/bearing/bearing.py

```diff
@@ -24,13 +24,12 @@ def median(self):
         else:
             return (sortedArray[index] + sortedArray[index + 1]) / 2.0
 
-
 g = {} # globals
 g['last_fix'] = None
-g['med_filter'] = MedianFilter(rospy.get_param('median_filter_size'))
+g['med_filter'] = MedianFilter(rospy.get_param('/buzzmobile/median_filter_size'))
 bearing_pub = rospy.Publisher('bearing', Float64, queue_size=1)
 
-MIN_FIX_DISTANCE = rospy.get_param('min_fix_distance')
+MIN_FIX_DISTANCE = rospy.get_param('/buzzmobile/min_fix_distance')
```

Comment:
They do currently namespace everything, but put all running nodes in the same namespace. You can still use the explicit form. Its just that since I'm spinning up nodes outside of the namespace, I *must* use the explicit form. That said, i think I can fix this elsewhere.